pkg/skaffold/build/bazel.go

```diff
@@ -30,6 +30,37 @@ import (
 	"github.com/pkg/errors"
 )
 
+func trimTarget(buildTarget string) string {
+
+	//TODO(r2d4): strip off leading //:, bad
+	trimedTarget := strings.TrimPrefix(buildTarget, "//")
```

Comment:
nit: typos. Could you rename these to "trimmed"?

Reply:
Done :)